src/Arcus.WebApi.Security/Authorization/AzureManagedIdentityAuthorizationFilter.cs

```diff
@@ -8,7 +8,7 @@ namespace Arcus.WebApi.Security.Authorization
 {
     public class AzureManagedIdentityAuthorizationFilter : IAsyncAuthorizationFilter
```

Comment:
The filter and reader should also be reviewed for XML docs. But obviously, not in this PR immediatly.